docker/min_settings.py

```diff
@@ -87,19 +87,19 @@
 
 BASE_ADDRESS = '{}:8000'.format(os.environ.get('HQ_PORT_8000_TCP_ADDR', 'localhost'))
 
-######## Email setup ########
+######## Email setup ######## noqa: E266
```

Comment:
What are these for?

Reply:
There was a linting issue where it wasn't happy about all the #'s. This tells the lint checker to ignore this comment